apps/login/src/app/(login)/verify/page.tsx

```diff
@@ -46,6 +46,8 @@ export default async function Page(props: { searchParams: Promise<any> }) {
 
   const doSend = invite !== "true";
 
+  const basePath = process.env.NEXT_PUBLIC_BASE_PATH ?? "";
+
```

Comment:
Why do we need this when we merge #380

Reply:
the code still has to apply the base path or am I missing something?

Reply:
sure, sry